tests/test_vendored_api.py

```
@@ -456,3 +475,301 @@ async def test_clients_get_all_explicit_limit_no_pagination() -> None:
+    """LAG get should raise ValueError when nothing is returned."""
+    client = _network_client()
+    client._get = AsyncMock(return_value=None)
+
+    with pytest.raises(ValueError, match="not found"):
+        await client.lags.get("site-1", "missing")
+
+
+async def test_mc_lag_domains_get_all() -> None:
+    """MC-LAG domain get_all should parse peers and local LAGs."""
+    client = _network_client()
+    client._get = AsyncMock(
```

Pylint (4.0.6)

[warning] Access to a protected member _get of a client class

(W0212)